AGENTS.md

```
@@ -46,4 +46,4 @@ Same command without the wrapper:
 npx skills add gannonh/skills --skill plan-build-verify --skill address-pr-comments -y
 ```
 
-Do not install the whole `gannonh/skills` pack. Cursor engineering execution uses the pstack plugin; do not npx-install `ps`. Skills already in `.agents/skills/` (`sandbox`, `readme`, and currently `ps`) stay as-is. This script only ensures the product OS.
\ No newline at end of file
+Skills are generated locally by this script and are not tracked.
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)